src/Orleans.Core/Runtime/Legacy/GrainTypeResolver.cs

```diff
@@ -1,36 +1,37 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Reflection;
 
 namespace Orleans.Runtime
 {
     internal interface IGrainTypeResolver
     {
         bool TryGetGrainClassData(Type grainInterfaceType, out GrainClassData implementation, string grainClassNamePrefix);
-        bool TryGetGrainClassData(int grainInterfaceId, out GrainClassData implementation, string grainClassNamePrefix);
-        bool TryGetGrainClassData(string grainImplementationClassName, out GrainClassData implementation);
         bool IsUnordered(int grainTypeCode);
-        string GetLoadedGrainAssemblies();
     }
 
     [Serializable]
     internal class GrainTypeResolver : IGrainTypeResolver
     {
         private readonly Dictionary<string, GrainInterfaceData> typeToInterfaceData;
+
+#pragma warning disable IDE0051 // Remove unused private members
+#pragma warning disable CS0169 // Remove unused private members
+        // Unused. Retained for serializer compatibility.
         private readonly Dictionary<int, GrainInterfaceData> table;
+
+        // Unused. Retained for serializer compatibility.
         private readonly HashSet<string> loadedGrainAsemblies;
+#pragma warning restore CS0169 // Remove unused private members
```

Comment:
are we still using the non-back compat serializer for the internal protocol?

Reply:
lol

Reply:
My hope is that we can change this in 4.0, since we now support network protocol negotiation and can hence add a new core serialization format.